models/tcp_route.go

```diff
@@ -13,6 +13,11 @@ type TcpRouteMapping struct {
 	TcpMappingEntity
 }
 
+// IMPORTANT!! when adding a new field here that is part of the unique index for
+//
+//	a tcp route, make sure to update not only the logic for Matches(),
+//	but also the SqlDb.readTcpRouteMapping() function's custom
```

Comment:
Is this meant to be `FindExistingTcpRouteMapping` ?

Reply:
yep. 😞